scripts/build-shaders.js

```diff
@@ -33,12 +33,15 @@ var data = data.split('\n').map(function(line) {
         } else {
             return params[1] + params[3] + 'p ' + params[4] + ' ' + params[5] + ' = u_' + params[5] + ';';
         }
+    } else if (line.match(pixel_ratio_regex)){
+        return '\n#define DEVICE_PIXEL_RATIO 2.0\n' + line;
```

Comment:
@mollymerp There are some shaders which use the `DEVICE_PIXEL_RATIO` outside of `#define` statements. I suggest following the precedent set in GL JS and adding this `#define` to the top of all shaders.